<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>755</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CSMS - HomePage</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>651</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MathJax_Vector-Bold</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Community Support Management System</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>351</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nimbus Roman [UKWN]</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>SEARCH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="members_page_btn">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>210</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code Light</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
selection-background-color: rgb(26, 95, 180);
background-color: rgb(153, 193, 241);</string>
    </property>
    <property name="text">
     <string>MEMBERS PAGE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MODULES</string>
    </property>
   </widget>
   <widget class="QLabel" name="members_count">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>210</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.714, y1:0.448864, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(38, 162, 105);</string>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>TOTAL MEMBERS: 58</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_from_prev">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>330</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code Light</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
selection-background-color: rgb(26, 95, 180);
background-color: rgb(16, 76, 102);</string>
    </property>
    <property name="text">
     <string>ADD MEMBER FROM PREVIOUS RECORDS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="register_member">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>330</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code Light</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
selection-background-color: rgb(26, 95, 180);
background-color: rgb(16, 76, 102);</string>
    </property>
    <property name="text">
     <string>REGISTER A NEW MEMBER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="all_reports">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>450</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code Light</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
selection-background-color: rgb(26, 95, 180);
background-color: rgb(16, 76, 102);</string>
    </property>
    <property name="text">
     <string>ALL REPORTS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="contrib_btn">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>450</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code Light</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
selection-background-color: rgb(26, 95, 180);
background-color: rgb(16, 76, 102);</string>
    </property>
    <property name="text">
     <string>CONTRIBUTIONS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loans_btn">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>570</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code Light</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
selection-background-color: rgb(26, 95, 180);
background-color: rgb(16, 76, 102);</string>
    </property>
    <property name="text">
     <string>DEBTS AND LOANS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="arrears_btn">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>570</y>
      <width>281</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code Light</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
selection-background-color: rgb(26, 95, 180);
background-color: rgb(16, 76, 102);</string>
    </property>
    <property name="text">
     <string>ARREARS</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>271</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>+254798989898</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>our_email@testmail.com</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Facebook Account</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>X</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="settings_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LM Roman Slanted 10</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
selection-background-color: rgb(26, 95, 180);
background-color: rgb(98, 160, 234);</string>
    </property>
    <property name="text">
     <string>Application Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="accounts_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LM Roman Slanted 12</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
selection-background-color: rgb(26, 95, 180);
background-color: rgb(16, 76, 102);</string>
    </property>
    <property name="text">
     <string>GROUP ACCOUNTS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>220</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>OPEN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="messages_btn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>341</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LM Roman Slanted 10</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
selection-background-color: rgb(26, 95, 180);
background-color: rgb(98, 160, 234);</string>
    </property>
    <property name="text">
     <string>Messages And Notifications</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>661</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>      Community Support Management System helps manage members,   meetings and track the group's activities.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OPEN GROUP'S WEBSITE AT https://www.our_group's_web.com</string>
    </property>
   </widget>
   <widget class="QPushButton" name="meetings_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>541</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Latin Modern Math</family>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>OPEN HISTORY: A RECORD OF PREVIOUS  MEETINGS AND EVENTS </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LM Roman Slanted 10</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ADMINSTRATION CONTACTS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tasks_btn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>410</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LM Roman Slanted 10</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
selection-background-color: rgb(26, 95, 180);
background-color: rgb(98, 160, 234);</string>
    </property>
    <property name="text">
     <string>TASKS AND REMINDERS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edit_combo_btn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>170</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>EDIT</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>671</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>170</y>
      <width>20</width>
      <height>561</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>16</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="on_going_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>80</y>
      <width>341</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MathJax_Main</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ON-GOING MEETING OR EVENT</string>
    </property>
   </widget>
   <widget class="QComboBox" name="on_going_combo">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>80</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>NONE</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew">
    <property name="title">
     <string>Record New</string>
    </property>
    <addaction name="actionRecordMeeting_btn"/>
    <addaction name="actionRecord_A_Burial_Event"/>
   </widget>
   <addaction name="menuNew"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRecord_Live_Meeting">
   <property name="text">
    <string>Record Live Meeting</string>
   </property>
  </action>
  <action name="actionRecordMeeting_btn">
   <property name="text">
    <string>Record A Meeting</string>
   </property>
  </action>
  <action name="actionRecord_A_Burial_Event">
   <property name="text">
    <string>Record An Event</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
